src/js/popup.js

```diff
@@ -180,6 +180,11 @@ function init() {
 
   $("#error").on("click", function() {
     $('#overlay').toggleClass('active');
+    // Show YouTube message on error reporting form
```

Comment:
This adds a scrollbar to the error reporting form. We could do some cleanup that will also take care of the scrollbar. Some suggestions:

- "Briefly describe the problem:" as the text area label, replacing both "Briefly describe the error below." and "Description". This should take care of the scrollbar
- Replace "Report an Error" with "Report broken site" (could reuse the button text? or does that not make sense in some languages? we could just reuse and see if there are any complaints)
- Replace "What's Wrong?" with "What's wrong?"
- Replace "Submit Error" with "Send report"

Reply:
Good suggestions, done!